scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -98,7 +98,6 @@ main_test
 scripts.build_test
 scripts.common_test
 scripts.concurrent_task_utils_test
-scripts.docstrings_checker_test
```

Comment:
Done, removed the file that I added tests to according to the suggestion from initial PR. (Suggestion for developer team: I think it was a bit unclear in the issue that this was a change that should be pushed to git so maybe make it a bit clearer :) )